include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +330,19 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     * Same as above, but using a vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
+     * <code>
+     * std::vector<Number> local_dof_values(n_dofs_per_cell);
```

Comment:
we usually use the name `dofs_per_cell`. But to make it really clear, you could write `cell->get_fe().dofs_per_cell` here